pkg/discovery/worker/group_discovery_policy_cvs.go

```diff
@@ -0,0 +1,108 @@
+package worker
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/turbonomic/kubeturbo/pkg/discovery/repository"
+	"github.com/turbonomic/kubeturbo/pkg/discovery/util"
+	"github.com/turbonomic/turbo-go-sdk/pkg/builder/group"
+	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
+)
+
+func buildContainerVerticalScalePolicy(
+	worker *k8sEntityGroupDiscoveryWorker,
+	policyBinding *repository.TurboPolicyBinding) (*proto.GroupDTO, error) {
+	// Resolve group members
+
+	resolvedIds, err := resolveCVSPolicyTargets(worker, policyBinding)
+	if err != nil {
```

Comment:
Do you want to log something about resolved IDs for a policy?

Reply:
will try to address this in https://vmturbo.atlassian.net/browse/OM-98560